src/main/java/org/junitpioneer/jupiter/PioneerException.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+public class PioneerException extends RuntimeException {
+
+	public PioneerException(String s, ReflectiveOperationException ex) {
```

Comment:
This was supposed to be `Exception` instead of `ReflectiveOperationException`. Oops.